stream_chat/client.py

```diff
@@ -321,3 +321,47 @@ def send_file(self, uri, url, name, user, content_type=None):
             headers=headers,
         )
         return self._parse_response(response)
+
+    def create_blocklist(self, name, words):
+        """
+        Create a blocklist
+
+        :param name: the name of the blocklist
+        :param words: list of blocked words
+        :return:
```

Comment:
This isn't strictly true as all these return the timings, but not sure if it needs to be documented.